### PR TITLE
chore: no code owners for auto-generated files

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
web team is assigned whenever auto-generated files are updated. This PR is to ignore these files to reduce noise.

## Test Plan
- Check if change is correct

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by dannysauer** [comment]
What generates these files outside of web's control?

If they're always essentially irrelevant, perhaps they should also be marked as `generated` in the .gitattributes file?

**Comment by dannysauer** [comment]
Oh, good, they *are* marked as generated already.
https://github.com/determined-ai/determined/blob/dde10f1b1aad2f3857402bbb231e7cc2c3ba57b3/.gitattributes#L8

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -24,4 +25,5 @@
 /e2e_tests/tests/experiment       @determined-ai/ml-sys
 /e2e_tests/tests/nightly          @determined-ai/ml-sys
 
-/webui @determined-ai/web
```
Comment (dannysauer) [inline .github/CODEOWNERS:27]: I kinda like the main directory and then exception near each other, similar to how /harness is done above.

Comment (keita-determined) [reply .github/CODEOWNERS:27]: ok. fix is pushed

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -24,4 +24,6 @@
...
\ No newline at end of file
+# Owned by @determined-ai/web except for generated files
+/webui    @determined-ai/web
+/webui/react/src/services/api-ts-sdk/**/*
```
Comment (ioga) [inline .github/CODEOWNERS:29]: fwiw I think this'll be equivalent to

```suggestion
/webui/react/src/services/api-ts-sdk/
```

**Review by ioga — APPROVED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (keita-determined) [reply .github/CODEOWNERS]: fixed

**Comment by keita-determined** [comment]
infra's approval is required to merge this

**Comment by dannysauer** [comment]
Oh, cool. Looks like you'll have a rebase conflict to fix in EE.

**Comment by keita-determined** [comment]
yeah will fix EE too

**Pull request merged**